### PR TITLE
Test/1693 component activated content menu

### PR DESCRIPTION
**Pull request by stevenburnell-moj**
JS test for component.
Only one tweak for original component code included.

### CODE REVIEW

File: test/component_activated_content_menu_test.js [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+const expect = require("chai").expect;
+
+describe("ContentMenu", function() {
+
+  const ContentMenu = require("../app/javascript/src/component_activated_content_menu");
+  const COMPONENT_CLASSNAME = "ContentMenu";
+  const CONTAINER_ID = "activated-content-menu-test-container-id";
+  const CONTAINER_CLASSNAME = "activated-content-menu-test-classname and-another-activated-menu-classname";
+  const ACTIVATOR_CLASSNAME = "activated-content-menu-activator-test-classname";
+  const ACTIVATOR_ID = "activated-content-menu-test-activator-id";
```
Comment (brenetic) [inline test/component_activated_content_menu_test.js:11]: i think this isn't used

File: test/component_activated_content_menu_test.js [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+  const ACTIVATOR_TEXT = "activated content menu activator";
+  const MENU_ID = "activated-content-menu-test-menu-id";
+  const TEST_SELECTION_ELEMENT_ID = "component-activated-content-menu-test-selection-event-element";
```
Comment (brenetic) [inline test/component_activated_content_menu_test.js:14]: i think this isn't used either

File: test/component_activated_content_menu_test.js (hunk 1 above)
Comment (stevenburnell-moj) [reply test/component_activated_content_menu_test.js]: Now removed

File: test/component_activated_content_menu_test.js (hunk 2 above)
Comment (stevenburnell-moj) [reply test/component_activated_content_menu_test.js]: Now removed

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**